specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachine.json

```diff
@@ -3206,6 +3206,11 @@
           },
           "description": "The virtual machine child extension resources."
         },
+        "managedBy": {
```

Comment:
new property should come at the end of the property list, for SDK generation purposes. 
So please put this one after "extendedLocation" few lines below